### PR TITLE
[RW-4252][risk=no] Change 'Any' to 'Any value' for attributes selections

### PR DESCRIPTION
**Pull request by dolbeew**
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ export const AttributesPage = withCurrentWorkspace() (
       const {node: {subtype}} = this.props;
       const{form, options} = this.state;
       if (!this.isMeasurement) {
-        options.unshift({label: 'Any', value: AttrName[AttrName.ANY]});
+        options.unshift({label: 'Any value', value: AttrName[AttrName.ANY]});
```
Comment (freemabd) [inline ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]:
```suggestion
        options.unshift({label: optionUtil.ANY.display, value: AttrName[AttrName.ANY]});
```

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx [hunk 2]
```diff
@@ -354,7 +354,7 @@ export const AttributesPage = withCurrentWorkspace() (
       let paramName;
       const attrs = [];
       if (form.exists) {
-        paramName = name + ' (Any)';
+        paramName = name + ' (Any value)';
```
Comment (freemabd) [inline ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]:
```suggestion
        paramName = name + ' (' + optionUtil.ANY.display + ')';
```

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx [hunk 3]
```diff
@@ -392,7 +392,7 @@ export const AttributesPage = withCurrentWorkspace() (
       form.num.filter(at => at.operator).forEach((attr, i) => {
         if (attr.operator === 'ANY') {
           if (i === 0) {
-            name += 'Any';
+            name += 'Any value';
```
Comment (freemabd) [inline ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]:
```suggestion
            name += optionUtil.ANY.display;
```

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx (hunk 1 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: Good idea, done.

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx (hunk 2 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: Done.

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx (hunk 3 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx]: Done.

**Pull request merged**